### PR TITLE
TTP remove version in metadata/add id in variants

### PR DESCRIPTION
**Pull request by bartimus-primed**

### CODE REVIEW

**Review by privateducky — APPROVED** [review]
@khyberspache can you second set of eyes this? 

two changes:
1. removal of version metadata
2. additional of an ID per variant

File: ttps/defense-evasion/b007f6e8-4a87-4440-8888-29ceab047d9b.yml [hunk 1]
```diff
@@ -52,3 +51,4 @@ platforms:
           wmic /namespace:\\root\microsoft\windows\defender path MSFT_MpPreference call set PUAProtection=0;
           wmic /namespace:\\root\microsoft\windows\defender path MSFT_MpPreference call set DisableAntiSpyware=TRUE;
           wmic /namespace:\\root\microsoft\windows\defender path MSFT_MpPreference call set TamperProtection=0;
+        id: 63bea820-7b32-418b-9eb6-a1b65c9c85c1
```
Comment (khyberspache) [inline ttps/defense-evasion/b007f6e8-4a87-4440-8888-29ceab047d9b.yml:54]: Same Q as above with the multiline block

File: ttps/discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml [hunk 2]
```diff
@@ -17,7 +16,8 @@ platforms:
     cmd:
       command: setspn.exe -Q */*
       variants:
-      - command: setspn.exe -T   #{domain.T1018} -Q */*
+      - command: setspn.exe -T
```
Comment (khyberspache) [inline ttps/discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml:19]: This is missing some ttp stuff

File: ttps/collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml [hunk 3]
```diff
@@ -37,3 +36,4 @@ platforms:
       variants:
       - command: |-
           [System.IO.Directory]::CreateDirectory("staged");
+        id: 9258543f-0082-45e3-93a6-02a770b96bee
```
[the remark this answers is not in the document]
Comment (khyberspache) [reply ttps/collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml:39]: Yeah I _think_ it should be okay, better to ask and double check

[the remark this answers is not in the document]
Comment (bartimus-primed) [reply ttps/collection/6469befa-748a-4b9c-a96d-f191fde47d89.yml:39]: yeah I am pretty sure its a GitHub issue, I can read the file in fine, and it is properly highlighted in VSCode

File: ttps/discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml (hunk 2 above)
Comment (bartimus-primed) [reply ttps/discovery/27b922c1-76ca-41c7-926f-1c36cabec122.yml]: fixed

**Comment by privateducky** [comment]
big $

**Review by privateducky — APPROVED** [review]

**Pull request merged**